jenkins/docker32/Dockerfile

```diff
@@ -62,7 +62,7 @@ RUN php composer-setup.php
 RUN mv composer.phar /usr/bin/composer
 RUN php -r "unlink('composer-setup.php');"
 RUN cd /tmp && \
-  git clone https://github.com/google/protobuf.git && \
+  git clone https://github.com/google/protobuf.git &&  \
```

Comment:
Why should we do that? Keep it here avoids download the composer stuff for each test.